### PR TITLE
Implemented "Monitoring iot data grafana" tutorial

### PR DESCRIPTION
**Pull request by leozz37**
Based on my [article](https://medium.com/google-cloud/monitoring-iot-devices-with-golang-google-cloud-platform-and-grafana-eb99cc36db4b) "Monitoring data from IoT devices with Golang, Google Cloud Platform and Grafana"  on Medium!

### CODE REVIEW

File: tutorials/monitoring-iot-data-grafana/index.md [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+### Register the device with IoT Core
+
+    gcloud iot devices create $DEVICE_ID \
+        --region=$REGION \
+        --registry=$REGISTRY \
+        --public-key="path=./ec_public.pem,type=es256"
+
+### Install Arduino libraries
+
+Install the “Google Cloud IoT Core JWT” and lwmMQTT from Joel Gaehwiler libraries on your Arduino IDE. These libraries are responsible for connecting,
```
Comment (ToddKopriva) [inline tutorials/monitoring-iot-data-grafana/index.md:179]: I think that we should give folks a bit more information about how to get these libraries.

Link here?

https://github.com/256dpi/arduino-mqtt

**Comment by ToddKopriva** [comment]
@leozz37 , I'm ready to merge this, but I do have a suggestion about giving the reader more information about the libraries that they need to install. See my review comment in the document.

File: tutorials/monitoring-iot-data-grafana/index.md (hunk 1 above)
Comment (leozz37) [reply tutorials/monitoring-iot-data-grafana/index.md:179]: These are the links for [Google Cloud IoT Core](https://github.com/GoogleCloudPlatform/google-cloud-iot-arduino) and this is the [lwMQTT lib](https://github.com/256dpi/arduino-mqtt).

We can also include this images:

Screen Shot 2020-10-29 at 01 09 33

Screen Shot 2020-10-29 at 01 09 06

Screen Shot 2020-10-29 at 01 10 58

Comment (ToddKopriva) [reply tutorials/monitoring-iot-data-grafana/index.md]: Thanks. I added the links.

**Review by ToddKopriva — APPROVED** [review]

**Comment by ToddKopriva** [comment]
Thanks again for the contribution, @leozz37 . I'll publish this with a batch of other documents within a couple of days.

**Comment by ToddKopriva** [comment]
https://cloud.google.com/community/tutorials/monitoring-iot-data-grafana

**Pull request merged**